packages/terra-form-select/src/SingleSelect.jsx

```diff
@@ -170,41 +166,40 @@ class SingleSelect extends React.Component {
 
   render() {
     const {
-      allowClear,
       children,
       defaultValue,
       onChange,
-      placeholder,
       required,
       value,
       intl,
+      isFilterStyle,
       ...otherProps
     } = this.props;
 
-    const defaultPlaceholder = intl.formatMessage({ id: 'Terra.form.select.defaultDisplay' });
-    const selectPlaceholder = placeholder === undefined ? defaultPlaceholder : placeholder;
-    let clearOptionDisplay;
+    let selectValue;
 
-    if (allowClear) {
-      if (selectPlaceholder.length === 0) {
-        clearOptionDisplay = defaultPlaceholder;
+    if (!isFilterStyle || this.state.value || value) {
+      selectValue = SelectUtil.value(this.props, this.state);
+    } else if (children) {
+      if (children[0].props.children) {
+        selectValue = children[0].props.children[0].props.value;
```

Comment:
I'm wary of these chained prop/array accessors being safe. I'd recommend borrowing/simplifying some of the accessors being used in the menu util. Children(OptGround/Options) have types that can be evaluated.

https://github.com/cerner/terra-core/blob/main/packages/terra-form-select/src/shared/_MenuUtil.js

Reply:
Removed the use of chained prop/array here https://github.com/cerner/terra-core/pull/3094/commits/5c0d19afda050f822e2d6e7fcd04abefa15886b6